src/main/java/pages/BasePage.java

```diff
@@ -0,0 +1,38 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class BasePage {
+    public WebDriver driver;
+    private By singInLink = By.className("authorization-link");
+    private By welcomeSpan = By.className("logged-in");
```

Comment:
It is not clear to me what this element refers to, perhaps it is the welcome message, if so it should be called:
`welcomeMessage`